code_samples/data_migration/src/Migrations/Step/ReplaceNameStepNormalizer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations\Step;
+
+use Ibexa\Platform\Contracts\Migration\Serializer\AbstractStepNormalizer;
+use Ibexa\Platform\Migration\ValueObject\Step\StepInterface;
+
+/**
+ * @extends \Ibexa\Platform\Contracts\Migration\Serializer\AbstractStepNormalizer<\App\Migrations\Step\ReplaceNameStep>
+ */
+final class ReplaceNameStepNormalizer extends AbstractStepNormalizer
+{
+    protected function normalizeStep(
+        StepInterface $object,
+        string $format = null,
+        array $context = []
+    ): array {
+        assert($object instanceof ReplaceNameStep);
+
+        return [
+            'replacement' => $object->getReplacement(),
+        ];
+    }
+
+    protected function denormalizeStep(
+        $data,
+        string $type,
+        string $format,
+        array $context = []
+    ): ReplaceNameStep {
+        return new ReplaceNameStep($data['replacement'] ?? null);
+    }
+
+    public function getHandledClassType(): string
+    {
+        return ReplaceNameStep::class;
+    }
+
+    public function getType(): string
+    {
+        return 'name';
```

Comment:
```suggestion
        return 'company_name';
```
would make it less likely to collide with a different migration type (it's unlikely as it is, but...)